src/offices/_single.html

```diff
@@ -1,50 +1,48 @@
-{% extends "layout-side-nav.html" %}
-{% import "_vars-offices.html" as vars with context %}
+{% extends 'layout-side-nav.html' %}
+{% import '_vars-offices.html' as vars with context %}
 
 {% block title -%}
     {{ office.title | safe if office.title }}
 {%- endblock %}
 
 {% block content_main_modifiers -%}
-    {{ super() }} content__flush-bottom office
+    {{ super() }}
+    content__flush-bottom
+    office
+    office_{{office.slug}}
 {%- endblock %}
 
 {% block content_main %}
 
-    {% import "macros/activity-snippets.html" as activity_snippets with context %}
-    {% set activities_feed = activity_snippets.render(office.tags if office.tags, include_date_flag=true, number_columns=2) %}
-
     {% if office.title %}
-        <h1>{{ office.title | safe }}</h1>
+    <h1>{{ office.title | safe }}</h1>
     {% endif %}
 
     {% if office.intro.text %}
+    {% set intro = office.intro %}
+    {% set show_subscription = intro.subscribe_form and intro.govdelivery_code %}
     <section class="block
                     block__flush-top
                     block__flush-bottom
                     content-l
                     content-l__main
                     content-l__large-gutters">
-        {% set intro = office.intro %}
-        {% set show_subscription = intro.subscribe_form and intro.govdelivery_code %}
-        <section class="content-l_col
-                        {% if show_subscription -%}
-                            content-l_col-1-2
-                        {% else -%}
-                            content-l_col-1
-                        {%- endif %}
-                        ">
-            <p class="h3">{{ intro.text | safe if intro.text }}</p>
-        </section>
+        <div class="content-l_col
+                    {{ 'content-l_col-1-1' if show_subscription else 'content-l_col-1' }}
+                    ">
+            {# Note this is a div to accommodate paragraph markup
```

Comment:
This code comment could possibly be removed. With tests set up for the office pages, they will fail if this gets turned back into a paragraph and a paragraph is passed in from the CMS. Also, there are other occurrences of this that are not documented in this way.